### ISSUE TITLE
Import new FPA_FOD_Agg.csv data

### ISSUE THREAD

**Issue by sawyerball** [cs481-ekh/s23-wildfire-wizards#28]
User Story #14 and #57

We were given new data by Yavar. We need to import this new data so that it is used by the website.

**Comment by sawyerball** [comment]
After spending more time on this issue, I have come to the realization that I need to either correct a misunderstanding or alter the import data script to: 

- Import files even if the table has already been created
- Import multiple files at once

{t:30}

**Comment by sawyerball** [comment]
Tried altering the create table script so that it would check if the table existed first before creating it. I think I just about got it working.

{t:70}

**Comment by sawyerball** [comment]
Altered the create table script to only create the table if it didn't exist. Then altered the create index portion to only create the index if it didn't already exist. This part I got help from the internet and it might need a little more looking into but it appears to be working as is right now.

{t:60}

**Comment by sawyerball** [comment]
Spent some time looking into an issue that wasn't in the code thinking it was. Quick fix.

{t:15}

**Comment by sawyerball** [comment]
While working on other task, I discovered a problem with my previous implementation. This turned out to be a bigger problem than I had previously anticipated. This will need more work in next sprint.

{t:160}

**Comment by sawyerball** [comment]
I've been working on this for many, many hours. I kept getting a "Decimal.InvalidOperation" error and could not for the life of me figure out where it was coming from as I could only logically make it out that I was dealing with ints. I seriously have no idea how much time I have spent sifting through code to find the issue. I only just know found what I THINK to be the issue but it is currently 2:30 in the morning and I need to sleep. I don't even know what to put for my time tag thing because I have seriously spent so much time on this in the last week and a half. Whatever I put is genuinely my best guess.

{t:900}
